packages/nimble-components/src/table-column/enum-text/group-header-view/index.ts

```diff
@@ -0,0 +1,52 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import { styles } from '../../text-base/group-header-view/styles';
+import { template } from '../../text-base/group-header-view/template';
+import type { TableColumnEnumColumnConfig } from '../../enum-base';
+import { TableColumnTextGroupHeaderViewBase } from '../../text-base/group-header-view';
+import type { TableFieldValue } from '../../../table/types';
+import type { MappingTextConfig } from '../../enum-base/models/mapping-text-config';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-enum-text-group-header-view': TableColumnEnumTextGroupHeaderView;
+    }
+}
+
+/**
+ * A group header view for enum columns
+ */
+export class TableColumnEnumTextGroupHeaderView extends TableColumnTextGroupHeaderViewBase<
+TableFieldValue,
+TableColumnEnumColumnConfig
+> {
+    private columnConfigChanged(): void {
+        this.updateText();
+    }
+
+    private groupHeaderValueChanged(): void {
+        this.updateText();
+    }
+
+    private updateText(): void {
+        const value = this.groupHeaderValue;
+        if (value === undefined || value === null) {
+            this.text = '';
+            return;
+        }
+
+        const config = this.columnConfig?.mappingConfigs.get(value);
+        this.text = config ? (config as MappingTextConfig).text : '';
```

Comment:
From what I can see, this logic appears to be the same as in the cell view but written differently. Can you pick one style and make them consistent so that it is more apparent that they are doing the same thing?